lib/parser/provenance.cc

```diff
@@ -241,6 +253,26 @@ const SourceFile *AllSources::GetSourceFile(
       origin.u);
 }
 
+std::optional<std::pair<int, int>> AllSources::GetSourcePosition(
+    Provenance prov) const {
+  const Origin &origin{MapToOrigin(prov)};
+  if (const auto inc = std::get_if<Inclusion>(&origin.u)) {
```

Comment:
When you use `const auto` for something that is a pointer, we always include the `*`. So:
`if (const auto *inc{...}) {`